src/DIRAC/FrameworkSystem/ConfigTemplate.cfg

```
@@ -5,8 +5,8 @@ APIs
   Auth
   {
     Port = 8000
-    # Allow download personal proxy
-    downloadablePersonalProxy = True
+    # Allow download personal proxy. By default: True
+    downloadablePersonalProxy = False
```

Shouldn't this match the default?
```suggestion
    downloadablePersonalProxy = True
```